client/src/components/protected/newGame/index.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
```

Comment:
I'm not a huge fan of using folders + index.js. Instead I find it clearer to just creat a file name newGame.js